tests/nix_tests/network.py

```diff
@@ -109,6 +109,31 @@ def setup_evmos(path, base_port, long_timeout_commit=False):
     yield from setup_custom_evmos(path, base_port, cfg)
 
 
+# for memiavl need to create the data/snapshots dir
+# for the nodes
+def create_snapshots_dir(path, base_port, config):
+    data_snapshots_dir = path / "evmos_9000-1" / "node0" / "data" / "snapshots"
+    os.makedirs(data_snapshots_dir, exist_ok=True)
+    data_snapshots_dir = path / "evmos_9000-1" / "node1" / "data" / "snapshots"
+    os.makedirs(data_snapshots_dir, exist_ok=True)
```

Comment:
is this always going to be 2 nodes? Maybe we can generalize this

```suggestion
def create_snapshots_dir(path, base_port, config, n_nodes=2):
    for idx in range(n_nodes):
        data_snapshots_dir = path / "evmos_9000-1" / f"node{idx}" / "data" / "snapshots"
        os.makedirs(data_snapshots_dir, exist_ok=True)
```